roles/splunk_deployer/tasks/push_apps_to_search_heads.yml

```diff
@@ -1,11 +1,4 @@
 ---
-- include_tasks: ../../../roles/splunk_common/tasks/find_installed_apps.yml
-
-- include_tasks: ../../../roles/splunk_common/tasks/copy_installed_apps.yml
-  vars:
-    apps: "{{ installed_apps }}"
-    dest: "{{ splunk.app_paths.shc }}"
```

Comment:
If we remove this, will this break existing functionality?

Reply:
I moved these tasks to main.yml so when executed from there the functionality would be the same.  If anyone outside this codebase calls this directly then maybe.